extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -243,13 +252,48 @@
   display: none !important;
 }
 
-.i-amphtml-story-new-page-control {
-  /* TODO(enriqe): this is a placehoder, replace when new icon is ready. */
-  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 48 48" fill="#FFFFFF"><path d="M0 0h48v48H0z" fill="none"/><path d="M22 34h4V22h-4v12zm2-30C12.95 4 4 12.95 4 24s8.95 20 20 20 20-8.95 20-20S35.05 4 24 4zm0 36c-8.82 0-16-7.18-16-16S15.18 8 24 8s16 7.18 16 16-7.18 16-16 16zm-2-22h4v-4h-4v4z"/></svg>') !important;
+.i-amphtml-story-has-new-page-notification-container {
+  display: flex !important;
+  position: absolute !important;
+  justify-content: center !important;
+  z-index: 100002 !important; /* On top of the progress bar. */
+  top: 32px !important;
+  right: 0 !important;
+  left: 0 !important;
+  opacity: 0 !important;
 }
 
-[i-amphtml-story-has-new-page] .i-amphtml-story-new-page-control {
-  display: block !important;
+.i-amphtml-story-has-new-page-text-wrapper {
+  background-color: rgba(32, 33, 37, 0.8) !important;
+  display: flex !important;
+  justify-content: center !important;
+  align-items: center !important;
+  padding-top: 4px !important;
+  padding-bottom: 4px !important;
+  padding-right: 4px !important;
+  border-radius: 5px !important;
+}
+
+.i-amphtml-story-has-new-page-text {
+  color: #fff !important;
+  font-size: 16px !important;
+  text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
+  font-weight: 700 !important;
+  margin-right: 8px !important;
+}
+
+.i-amphtml-story-has-new-page-circle-icon {
+	background: rgba(3, 255, 160, 1) !important;
+	border-radius: 50% !important;
```

Comment:
This should have a `4px` border at `rgba(3, 255, 160, 0.5)`

Reply:
Did you mean to comment this on the line below of the box-shadow, which represents the outer circle?

Reply:
If so, according to the spec the outer circle is supposed to be 10px in width, while the inner circle is 6px in width.

And if I understand correctly, the box-shadow `spread` property specifies the *radius* of the circle.  Meaning that if we want a 10px-wide circle, then the want to have a 5px radius. And since the inner circle is a 6px-wide circle (with a radius of 3px), then the box-shadow `spread` property should be 2px to add up to a total radius of 5px, giving us a 10px-wide outer circle.